extensions/objects/models/objects.py

```diff
@@ -122,6 +122,26 @@ class UnicodeString(BaseObject):
     }
 
 
+class SubtitledUnicode(BaseObject):
+    """SubtitledUnicode class."""
+
+    description = 'An Subtitled string.'
+    default_value = {'unicode_str': '', 'content_id': None}
+
+    SCHEMA = {
+        'type': 'dict',
+        'properties': [{
+            'name': 'unicode_str',
+            'schema': UnicodeString.SCHEMA
+        }, {
+            'name': 'content_id',
+            'schema': {
+                'type': 'optional_unicode'
```

Comment:
I think a better type would be unicode_or_none. Otherwise this sort of implies that the entire field is optional.